examples/demo/src/App.tsx

```diff
@@ -26,19 +26,17 @@ const searchClient = algoliasearch(appId, apiKey);
 const recommendClient = algoliarecommend(appId, apiKey);
 
 insights('init', { appId, apiKey });
+insights('setUserToken', 'user-token-1');
 
-function RecommendedItem({ item }) {
-  return <Hit hit={item} insights={insights} />;
-}
-
-function BundleItem({ item }) {
+function BundleItem({ item, onSelect }: BundleItemProps) {
```

Comment:
We can make this component generic.

Reply:
Sure, done here: https://github.com/algolia/recommend/pull/48/commits/59023d00c034a53f8adfcd29f386a7181c790755